modules/persistent-storage-csi-migration-enable.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+[id="persistent-storage-csi-migration-enable_{context}"]
+= Enabling CSI automatic migration
+
+[NOTE]
+====
+Enabling CSI automatic migration drains, and then restarts, all nodes in the cluster in sequence. This might take some time.
+====
+
+.Procedure
+
+* Enable feature gates (see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling features using feature gates]).
++
+[IMPORTANT]
+====
+After turning Technology Preview features on using feature gates, they cannot be turned off and cluster upgrades are prevented.
```

Comment:
I'm using the suggested change feature here, but be sure not to use the button to accept the changes. Just moving "on" closer to "turning", and I'd either break up into two sentences, or add a semicolon.

```suggestion
After turning on Technology Preview features by using feature gates, they cannot be turned off. As a result, cluster upgrades are prevented.
```